node/p2p_block_exchange.md

```diff
@@ -0,0 +1,101 @@
+# P2P Block Exchange
```

Comment:
Did are align that it was Block Sync? and not P2P as part of the title?

Reply:
Would apply the resolution here to the rest of the doc.

Reply:
Applied